src/pybind/mgr/dashboard/frontend/src/app/shared/components/card/card.component.ts

```diff
@@ -14,7 +14,11 @@ export class CardComponent {
   @Input()
   cardType: string = '';
   @Input()
-  removeBorder = false;
+  borderClass = '';
   @Input()
-  shadow = false;
+  shadowClass = 'shadow-sm';
+  @Input()
+  heightClass = '';
+  @Input()
+  bgColorClass = '';
```

Comment:
This is more of a philosophical discussion, but IMHO this is mixing presentational and behavioral concerns. Components (and their inputs) should care about what and how they do stuff, not what they look like.

That means that instead of having a parent component telling its children component how they have to look, it should just state what they need to do, and either:
* component look & feel is controlled externally (out-of-band) via ad-hoc CSS classes/styles.
* components adapt themselves their look & feel according to functional/controlled inputs (I'd prefer the `removeBorder`, or even better, just `border`, even if they cope with presentational concerns, than letting it pass a CSS class).

Classes shouldn't allow input arguments that can potentially break them, or alter their look & feel in undefined ways (and passing class names allows both things).